.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
I see .idea/ added to .gitignore which is great, but I am still seeing these files in the PR?

Reply:
Good point.

Warning: a git n00b is in the wild. One of the issues with git that I have had for a while is that of what you speak: it automatically continues to track files that are no longer part of a release. :-( Good as time as ever to go back and see if I can get git/GitHub to totally forget about these files particularly as they were never part of a release.

Agree: with the exception of .gitignore, the 76er fork should not have any additional files than the master (convenient way of putting it).

Reply:
Believe both the .idea directory as well config.php have been deleted from the repo's complete history, i.e., there is no evidence of them the directory file from ever existing. 

Recorded here so I can come back to it some years from now: 
git filter-branch --force --index-filter \
  "git rm --cached --ignore-unmatch PATH-TO-YOUR-FILE-WITH-SENSITIVE-DATA" \
  --prune-empty --tag-name-filter cat -- --all
[GitHub](https://help.github.com/en/github/authenticating-to-github/removing-sensitive-data-from-a-repository) from the CLI.

`--prune-empty` and `--tag-name-filter` threw errors so deleted. As a directory was being deleted, use recursion:
`git filter-branch --force --index-filter "git rm --cached --ignore-unmatch .idea -r cat -- --all` to remove .idea directory and `git filter-branch --force --index-filter "git rm --cached --ignore-unmatch config.php -r cat -- --all` to remove config.php . 

Finally `git push origin --force --all` to push the changes back up on the repo.